grid-cli/cmd/get_contract.go

```diff
@@ -50,4 +64,5 @@ var getContractCmd = &cobra.Command{
 
 func init() {
 	getCmd.AddCommand(getContractCmd)
+	getContractCmd.Flags().Bool("no-color", false, "disable output styling")
```

Comment:
why not adding it to the parent command instead of adding it for every subcommand?